### DISCUSSION TITLE
[SUGGESTION] Improvements in the Schema editor - Untangling lines of links

### DISCUSSION THREAD

**Discussion by StasKoun2** [TiagoSilvaPereira/vemto-issues#147] [Ideas & Suggestions]
Relationships in the schema-data editor are very confusing when there are a large number of interconnected models.
Therefore, I suggest a way to "untangle the tangle of connections", which, in my opinion, will significantly improve ergonomics.
It should be possible to select and break the connection line and get the corresponding instances of the target model at each end of the break. (see figure) 
And to undo the break, if you delete one of the model instances, both instances with their connection lines should be deleted and the original connection should be restored.

![nnn_2-1](https://user-images.githubusercontent.com/80303996/110420367-b9df4400-80cd-11eb-9d99-8d233e006be6.jpg)

![nnn_1](https://user-images.githubusercontent.com/80303996/110419812-9d8ed780-80cc-11eb-8d16-0f0673255fc6.jpg)

**Comment by fredyns** [comment]
i do agree that relationship line are messy & need improvement.
this GUI Schema Editor is the sole reason i migrate to laravel. it helps me alot in designing database.

i hardly recognize the relationship context if the line goes like this.
![schema](https://user-images.githubusercontent.com/4033019/138561832-12127ff4-b389-4562-a2f0-83f9886f046b.png)

thus i suggesting 2 solution:
1. easy way: ability to change relationship color
2. ideal way: multiple schema editor page like WorkBench

i believe this could be a huge impact in designing database.

**Reply by jgulledge19** [reply]
FYI #236 and #292 are related
